doc.go

```diff
@@ -234,6 +234,12 @@ type Config struct {
 	// EnableProxyProtocol controls the proxy protocol
 	EnableProxyProtocol bool `json:"enabled-proxy-protocol" yaml:"enabled-proxy-protocol" usage:"enable proxy protocol"`
 
+	// UseLetsEncrypt controls if we should use letsencrypt to retrieve certificates
+	UseLetsEncrypt bool `json:"use-letsencrypt" yaml:"use-letsencrypt" usage:"use letsencrypt for certificates"`
+
+	// LetsEncryptCacheDir is the path to store letsencrypt certificates
+	LetsEncryptCacheDir string `json:"letsencrypt-cache-dir" yaml:"lets-encrypt-cache-dir" usage:"path where cached letsencrypt certificates are stored"`
```

Comment:
I have also use-letsencrypt, should we go for all letsencrypt or lets-encrypt? use-lets-encrypt seems a bit strange.